doc/stdlib/libc.rst

```diff
@@ -40,7 +40,10 @@
 
 .. function:: time(t::TmStruct)
 
-   Converts a ``TmStruct`` struct to a number of seconds since the epoch.
```

Comment:
Ok, ignore me, I misread the `Libc.time` docstring and it's actually OK – this is just the method-clobbering issue again but the data isn't lost.